### PR TITLE
Linters

### PR DESCRIPTION
**Pull request by amsowie**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,4 +44,21 @@ jobs:
...
+            echo $DJANGO_VERSIONS | tr ";" "\n" | xargs -L 1 -I % tox -e dj%
+  test-py36:
+    <<: *test-template
+    docker:
+      - image: circleci/python:3.6
+    environment:
+      - DJANGO_VERSIONS: "111;20"
+workflows:
+  version: 2
+  test-py27-and-test-py36-and-lintcheck:
```
Comment (RevolutionTech) [inline .circleci/config.yml]: This is great! Can we rename this to something more concise? `check-and-test` or even just `test` would do.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[MASTER]
+
+# List of plugins (as comma separated values of python modules names) to load,
+# usually to register additional checkers.
+load-plugins=sentry_stack_checker,
```
Comment (RevolutionTech) [inline .pylintrc]: You may need to also add the `trailing_comma` and `django_redundant_all` checkers too.

Comment (amsowie) [reply .pylintrc]: I removed those because of no module found errors. The errors come back when I re-add them. Suggestions?

Comment (RevolutionTech) [reply .pylintrc]: Taking a look at your `.circleci/config.yml` I noticed that you're using the `circleci/python:2.7` docker container but your `PYTHONPATH` environment variable points to a `python3.6` directory, which is not where `isc-style-guide` is installed. That may be the reason why those modules cannot be found. You could fix this either by updating the `PYTHONPATH` or changing which docker container you're using.

Comment (amsowie) [reply .pylintrc]: That worked. Thanks!

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+import-order-style = appnexus
+application-import-names = flavourdetect
+exclude = .svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg
```
Comment (RevolutionTech) [inline .flake8]: Is this line necessary? I think that's pretty close to the default for `--exclude`.

**Pull request merged**